textacy/tm/topic_model.py

```diff
@@ -292,3 +300,135 @@ def topic_weights(self, doc_topic_matrix):
+                     save=False):
+        """
+        Make a "termite" plot for assessing topic models using a tabular layout
+        to promote comparison of terms both within and across topics.
+
+        Args:
+            doc_term_matrix (``np.ndarray``-like or sparse matrix): corpus
+                represented as a document-term matrix with shape (n_docs, n_terms);
+                may have tf- or tfidf-weighting
+            id2term (list(str) or dict): object that returns the term string corresponding
+                to term id ``i`` through ``id2term[i]``; could be a list of strings
+                where the index represents the term id, such as that returned by
+                ``sklearn.feature_extraction.text.CountVectorizer.get_feature_names()``,
+                or a mapping of term id: term string
+            topics (seq(int) or int, optional): topic(s) to include in termite plot;
+                if -1, all topics are included
+            sort_topics_by ({'index', 'weight'}, optional):
+            highlight_topics (seq(int) or int, optional): indices for up to 6 topics
+                to visually highlight in the plot with contrasting colors
+            n_terms (int, optional): number of top terms to include in termite plot
+            rank_terms_by ({'topic_weight', 'corpus_weight'}, optional): value used
+                to rank terms; the top-ranked ``n_terms`` are included in the plot
+            sort_terms_by ({'seriation', 'weight', 'index', 'alphabetical'}, optional):
+                method used to vertically sort the selected top ``n_terms`` terms;
+                the default ("seriation") groups similar terms together, which
+                facilitates cross-topic assessment
+            save (str, optional): give the full /path/to/fname on disk to save figure
+
+        Returns:
+            ``matplotlib.axes.Axes.axis``: axis on which termite plot is plotted
+
+        Raises:
+            ValueError: if more than 6 topics are selected for highlighting, or
+                an invalid value is passed for the sort_topics_by, rank_terms_by,
+                and/or sort_terms_by params
+
+        References:
+            .. Chuang, Jason, Christopher D. Manning, and Jeffrey Heer. "Termite:
+                Visualization techniques for assessing textual topic models."
+                Proceedings of the International Working Conference on Advanced
+                Visual Interfaces. ACM, 2012.
+            .. for sorting by "seriation", see https://arxiv.org/abs/1406.5370
+
+        .. seealso:: :func:`viz.termite_plot <textacy.viz.termite.termite_plot>`
+
+        TODO: `rank_terms_by` other metrics, e.g. topic salience or relevance
+        """
+        if highlight_topics is not None:
+            if isinstance(highlight_topics, int):
+                highlight_topics = (highlight_topics,)
+            elif len(highlight_topics) > 6:
```

Comment:
I only had good light/dark pairings for 6 colors. :wink: But I also wanted to limit the amount of visual complexity added to the plot — they're meant to highlight key aspects, not _all_ aspects.